randnum.c

```diff
@@ -0,0 +1,28 @@
+//Bettina Benitez
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int main() {
+	srand(time(0));
+	int r = rand () % 11; //picks a random number between 0 and 10 (11 numbers)
```

Comment:
You were incredibly close! The range needed to be from 1 to 10, but it included 0. To fix this, you could have done int r = rand()%10 +1; Either way, you were on the right track to doing this efficiently. Nice commenting, also!